[DOCS] Updates advanced settings doc (#93968)
elastic/kibana: docs/management/advanced-options.asciidoc

number of bars.

[[histogram-maxbars]]`histogram:maxBars`::

> Limits the density of date and number histograms across Kibana

Reviewer: ```suggestion
To improve performance, limits the density of date and number histograms across {kib}
```

> for better performance using a test query. If the test query would too many buckets,

Reviewer: ```suggestion
using a test query. When the test query contains too many buckets,
```

> the interval between buckets will be increased. This setting applies separately

Reviewer: ```suggestion
the interval between buckets increases. This setting applies separately
```

> to each histogram aggregation, and does not apply to other types of aggregation.

Reviewer: ```suggestion
to each histogram aggregation, and does not apply to other types of aggregations.
```
